ui/guest/src/react/ExperienceBuilder.tsx

```diff
@@ -107,6 +108,8 @@ import { setJwt } from '@craftercms/studio-ui/utils/auth';
 import { SHARED_WORKER_NAME } from '@craftercms/studio-ui/utils/constants';
 import useUnmount from '@craftercms/studio-ui/hooks/useUnmount';
 import { DeepPartial } from '@craftercms/studio-ui/models/DeepPartial';
+import { emitSystemEvent, openSiteSocket } from '@craftercms/studio-ui/state/actions/system';
```

Comment:
`openSiteSocket` not used
```suggestion
import { emitSystemEvent } from '@craftercms/studio-ui/state/actions/system';
```